nixpkgs/github.json

```diff
@@ -1,7 +1,7 @@
 {
-  "owner": "nixos",
+  "owner": "WebGHC",
   "repo": "nixpkgs",
-  "branch": "release-19.09",
-  "rev": "49550f29cd9d6ed27f4a76ba3c3fe30edf882eb7",
-  "sha256": "003fpkrcc5mw33q670q4z865zpr430cllbxhyzj8k9p4hbc8yqda"
+  "branch": "wasm-r-p-1909-integ",
```

Comment:
what's this? can this get merged to a main branch?

Reply:
There are some changes like (`lib/systems/doubles.nix`) which cannot be put in upstream, neither can they be done via overlay, so we will have to use a fork.

Reply:
The `lib/systems/doubles.nix` change looked good to upstream for me?

Reply:
Let's get a PR up make a ticket to check back on it.

Reply:
@Ericson2314 yes, now I realized that it should be possible to upstream that, even though the cross-compilation to wasm will still happen by overlays.